src/main/webapp/app/shared/components/code-button/code-button.component.html

```diff
@@ -39,28 +39,40 @@ <h5>{{ cloneHeadline | artemisTranslate }}</h5>
                 </div>
             }
             <!-- 'url-box-remove-line-left': Remove round edges and edge on the left side (due to dropdown to select ssh or https)
-           `url-box-remove-line-right`: Remove round edges and edge on the right side (not available for LocalVC) -->
+           `url-box-remove-line-right`: Remove round edges and edge on the right side -->
             <pre
                 class="clone-url"
                 [ngClass]="{
-                    'url-box-remove-line-left': sshEnabled && ((useSsh && !localVCEnabled) || !useSsh),
-                    'url-box-remove-line-right': !localVCEnabled && !(sshEnabled && ((useSsh && !localVCEnabled) || !useSsh)),
+                    'url-box-remove-line-left': sshEnabled,
+                    'url-box-remove-line-right': !localVCEnabled || (!useSsh && !!routerLinkForRepositoryView),
                 }"
                 [cdkCopyToClipboard]="getHttpOrSshRepositoryUri(false)"
                 (cdkCopyToClipboardCopied)="onCopyFinished($event)"
                 >{{ getHttpOrSshRepositoryUri() }} </pre
             >
-            @if (!useSsh && !localVCEnabled) {
+            @if (!useSsh) {
                 <div class="d-flex align-items-center">
-                    <a
-                        style="flex: 1"
-                        class="btn btn-secondary btn-sm me-2 open-repository-button"
-                        [href]="getHttpRepositoryUri() | safeUrl"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        <fa-icon [icon]="faExternalLink" />
-                    </a>
+                    @if (!localVCEnabled) {
```

Comment:
I'm open to suggestions that reduce the redundancy here - unfortunately, I cannot attach a `href` and `routerLink` to the same anchor, as even `[routerLink]="undefined"` will overwrite the `href`